src/test/lib/fixed_string_test.cpp

```diff
@@ -0,0 +1,92 @@
+#include <memory>
```

Comment:
In my branch, I commented most of this file out, but the tests should be adapted to the new constructor interface